Prog/2023/Housing Strikeforce Sales Tabs.sas

```diff
@@ -0,0 +1,200 @@
+/**************************************************************************
+ Program:  Housing Strikeforce Sales Tabs.sas
+ Library:  Requests
+ Project:  Urban-Greater DC
+ Author:   Rob Pitingolo
+ Created:  06/29/22
+ Version:  SAS 9.4
+ Environment:  Windows
+ 
+ Description: Uses DC sales data to output tabulations about sales, median
+				prices and price appreciation. 
+
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( Requests )
+%DCData_lib( realprop );
+%DCData_lib( ACS );
+%DCData_lib( Census );
+
+%let inflation_year = 2022;
```

Comment:
Since we only have income in 2021 $ - should we keep 2021 as the reference year? I think that makes the most sense.